starlite/utils/module_loading.py

```diff
@@ -0,0 +1,32 @@
+import sys
+from importlib import import_module
+
+
+def import_string(dotted_path):
+    """
+    Import a dotted module path and return the attribute/class designated by the
+    last name in the path. Raise ImportError if the import failed.
+
+    Based on Django's module_loading code
```

Comment:
```suggestion
```

Reply:
unless you literally ported the code from django, in which case you need a comment that references the original. Otherwise, if its just inspiration, no need for this comment.

Reply:
It's very close to what Django has.  Once I remove the Walrus operator, it maybe sufficiently different though.